liftwizard-maven-build/liftwizard-minimal-parent/pom.xml

```
@@ -242,7 +242,7 @@
             <plugin>
                 <groupId>org.openrewrite.maven</groupId>
                 <artifactId>rewrite-maven-plugin</artifactId>
-                <version>5.41.0</version>
+                <version>5.42.0</version>
```

Curious to hear if you see improvement with this patch release.
```suggestion
                <version>5.42.1</version>
```

---

There was a bit of a delay in getting 5.42.1 out, but it should become available shortly:
https://github.com/openrewrite/rewrite-maven-plugin/releases/tag/v5.42.1